articles/storage/common/azure-defender-storage-configure.md

```diff
@@ -93,21 +93,21 @@ Use an Azure Policy to enable Microsoft Defender for Cloud across storage accoun
 
 ### [PowerShell](#tab/azure-powershell)
 
-To enable Microsoft Defender for Storage for a storage account via PowerShell, first make sure you have installed the [Az.Security](https://www.powershellgallery.com/packages/Az.Security) module. Next, call the [Enable-AzSecurityAdvancedThreatProtection](/powershell/module/az.security/enable-azsecurityadvancedthreatprotection) command. Remember to replace values in angle brackets with your own values:
+To enable Microsoft Defender for Storage for a storage account via PowerShell, first make sure you have installed the [Az.Security](https://www.powershellgallery.com/packages/Az.Security) module. Next, call the [Enable-AzSecurityAdvancedThreatProtection](https://github.com/Azure/azure-powershell/blob/main/src/Security/Security/help/Enable-AzSecurityAdvancedThreatProtection.md) command. Remember to replace values in angle brackets with your own values:
```

Comment:
We need to use relative links, and also we don't want to link to GitHub here. The original link is correct. Please revert.